openshift/openshift-virtualization.rst

```diff
@@ -100,6 +100,29 @@ Prerequisites
 
 * If planning to use NVIDIA vGPU, SR-IOV must be enabled in the BIOS if your GPUs are based on the NVIDIA Ampere architecture or later. Refer to the `NVIDIA vGPU Documentation <https://docs.nvidia.com/grid/latest/grid-vgpu-user-guide/index.html#prereqs-vgpu>`_ to ensure you have met all of the prerequisites for using NVIDIA vGPU.
 
+***********************************************************
+Configure NVIDIA GPU Operator with OpenShift Virtualization
+***********************************************************
+
+After configuring the :ref:`prerequisites<prerequisites>`, the high level workflow for using the NVIDIA GPU Operator with OpenShift Virtualization is as follows:
+
+* :ref:`Enable the IOMMU driver <enable-iommu-driver>`.
```

Comment:
@cdesiniotis Is this step always required? Or only in certain cases? Just wanted to confirm before adding it here